src/utils/convertPipes/PlaceholderObjectBuilder.js

```diff
@@ -21,7 +21,7 @@ const TRANSFORM_MAP = {
   Number: { format: FORMAT_NUMBER, transformKey: NUMBER_TRANSFORMATION },
   Date: { format: FORMAT_DATE, transformKey: DATE_TRANSFORMATION },
   DateRange: { format: FORMAT_DATE, transformKey: DATE_TRANSFORMATION },
-  Unit: { format: FORMAT_UNIT, transformKey: NUMBER_TRANSFORMATION },
+  // Unit: { format: FORMAT_UNIT, transformKey: NUMBER_TRANSFORMATION },
```

Comment:
Just the commented out code here 👍 .

Reply:
Leaving it in until we know whats happening with unit